data-explorer/kusto/management/workload-groups.md

```diff
@@ -9,14 +9,14 @@ ms.date: 04/30/2021
 
 A workload group serves as a container for requests (queries, commands) that have similar classification criteria. Workload groups and [workload group policies](#workload-group-policies) are a means of resource governance for incoming requests to the cluster, and allow aggregate monitoring of the requests. When a request's execution begins, the request is classified and assigned to a specific workload group. Then, the request runs using the policies assigned to the workload group.
 
-Workload groups are defined at the cluster level. Up to 10 custom workload groups may be defined in addition to the two built-in workload groups.
+Workload groups are defined at the cluster level. Up to 10 custom workload groups may be defined in addition to the three built-in workload groups.
 
 > [!NOTE]
 > Requests that aren't queries or control commands aren't included in the scope of workload groups. For example: streaming ingestion requests.
 
 ## Built-in workload groups
 
-There are two pre-defined workload groups: [the `internal` workload group](#internal-workload-group) and [the `default` workload group](#default-workload-group).
+There are three pre-defined workload groups: [the `internal` workload group](#internal-workload-group), [the `default` workload group](#default-workload-group) and [the `materialized-views` workload group](#materialized-views-workload-group).
```

Comment:
the name is actually `$materialized-views`, right?

Reply:
yes